butterknife/src/test/java/butterknife/OnTouchTest.java

```diff
@@ -33,6 +34,7 @@
         + "  private Test target;\n"
         + "  private View view1;\n"
         + "  @UiThread\n"
+        + "  @SuppressLint(\"ClickableViewAccessibility\")\n"
```

Comment:
Unfortunately, it's not that easy without extending `ListenerMethod` or without including another hard-coded check for `OnTouch` in `BindingSet#addMethodBindings(..)`. What do you think?